app/src/main/java/protect/card_locker/LoyaltyCardViewActivity.java

```diff
@@ -221,6 +224,10 @@ public boolean onOptionsItemSelected(MenuItem item)
                 finish();
                 break;
 
+            case R.id.action_share:
+                importURIHelper.startShareIntent(importURIHelper.toUri(loyaltyCard));
```

Comment:
The two places that toUri is used it is passed directly to startShareIntent. Why not have startShareIntent take the card directly, and not expose toUri?